testing/execution.ts

```diff
@@ -1199,6 +1210,70 @@ async function executeSyncFormulaWithContinuations({
   return {result, chainedCommandResults, completion};
 }
 
+async function executeGetPermissionsFormulaWithContinuations({
+  formulaSpecification,
+  params,
+  manifest,
+  executionContext,
```

Comment:
Super nit, I think it would be safer if you aliased this input, using `executionContext: itemsExecutionContext` or something. Then you can call your copy `executionContext`. With the default naming, if you forget to use the copy you'll accidentally pass through this source context and get the pollution issue you're worried about.